script.sh

```diff
@@ -208,14 +210,45 @@ RUN echo "deb-src ${TRAVIS_DEBIAN_MIRROR} experimental main" >> /etc/apt/sources
 EOF
 fi
 
+EXTRA_PACKAGES=""
+
+case "${TRAVIS_DEBIAN_EXTRA_REPOSITORY}" in
+	https:*)
+		EXTRA_PACKAGES="apt-transport-https"
```

Comment:
Even though `$EXTRA_PACKAGES` is always an empty string here? I understand it's preferable to do it in case someone later puts new code that modifies `EXTRA_PACKAGES` before the case block, but I don't see why I "need to do it" - I tested the script and it works just fine. (Just being curious, I'm updating the code anyway.)